sources/odoo.conf

```diff
@@ -4,7 +4,7 @@
 ; --------------------
 ; change this ASAP!!!!
 ;---------------------
-admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+admin_passwd = _ADMIN_PASSWORD_
 
 ; set the default timezone for openerp interface
 timezone = Europe/Paris
```

Comment:
@elicoidal we can use sed in https://github.com/Elico-Corp/odoo-docker-china to replace the TZ in the conf file. But not in the generic "worldwide" Odoo image